emails/views.py

```diff
@@ -1557,11 +1544,7 @@ def _handle_complaint(message_json: AWS_SNSMessageJSON) -> HttpResponse:
     * complaint_user_agent - identifies the client used to file the complaint
     * complaint_extra - Extra data from complainedRecipients data, if any
     * domain - User's domain, if an address was given
-
-    Emits a legacy log "complaint_received", with data:
-    * recipient_domains: list of extracted user domains
-    * subtype: 'onaccounsuppressionlist', or 'none'
-    * feedback: feedback from ISP or 'none'
+    * fxa_id - The Mozilla account ID of the user
```

Comment:
```suggestion
    * fxa_id - The Mozilla account (previously known as Firefox Accounts) ID of the user
```
(suggestion, non-blocking) In case Relay survives long enough and Firefox Accounts have long been dropped for Mozilla Accounts.

Reply:
and in 2027 we can change it to

```suggestion
    * fxa_id - The Firefox account (previously known as Mozilla account) ID of the user
```